lib/wsdl.js

```diff
@@ -883,6 +885,9 @@ ElementElement.prototype.description = function(definitions, xmlns) {
 
         if (this.$ref) {
           element = elem;
+          if (arrayHint) {
```

Comment:
As `arrayHint` is guaranteed to be set correctly after its initialization, I would say it's unneccessary to check for `arrayHint` to be truthy before setting it. Just do `element.arrayHint = arrayHint;`  which should be save as it's either `false` or `true` :smiley: